File: Exercise14-ExamPreparation/Kanban/PHP/src/AppBundle/Entity/Task.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;


/**
 * @ORM\Table(name="tasks")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\TaskRepository")
 */
class Task
{
	/**
	 * @var int
	 * @ORM\Column(name="id", type="integer")
	 * @ORM\Id
	 * @ORM\GeneratedValue(strategy="AUTO")
	 */
	private $id;

	/**
	 * @return int
	 */
	public function getId()
	{
		return $this->id;
	}

	/**
	 * @param int $id
	 */
	public function setId($id)
	{
		$this->id = $id;
	}

	/**
	 * @var string
	 * @Assert\NotBlank()
	 * @Assert\Length(min = 1, max = 64,
	 *     minMessage = "Title cannot be blank",
	 *     maxMessage = "Title too long! No more than {{ limit }} characters allowed")
	 * @ORM\Column(name="title", type="string", length=64)
	 */
	private $title;

	/**
	 * @return string
	 */
	public function getTitle()
	{
		return $this->title;
	}

	/**
	 * @param string $title
	 * @return Task
	 */
	public function setTitle($title)
	{
		$this->title = $title;
	}

	/**
	 * @var string
	 * @Assert\NotBlank()
	 * @ORM\Column(name="status", type="string")
	 */
	private $status;

	/**
	 * @return string
	 */
	public function getStatus()
	{
		return $this->status;
	}

	/**
	 * @param string $status
	 * @return Task
	 */
	public function setStatus($status)
	{
		$this->status = $status;
	}
}

File: Exercise14-ExamPreparation/Kanban/JAVA/src/main/java/teistermask/entity/Task.java
package teistermask.entity;

import javax.persistence.*;

@Entity
@Table(name = "tasks")
public class Task {

    private Integer id;

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    private String title;

    @Column(nullable = false)
    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    private String status;

    @Column(nullable = false)
    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public Task(String title, String status) {
        this.title = title;
        this.status = status;
    }

    public Task() {    }
}
